components/common/src/ome/api/Search.java

```diff
@@ -158,7 +158,7 @@
 
     /**
      * Returns the current case sensitivity setting. If
-     * {@link #setCaseSentivice()} has not been called, the
+     * {@link #setCaseSentivice(boolean)} has not been called, the
```

Comment:
That's what I also assumed first but this is the current name of the method. We can definitely rename it (follow-up PR?) but I have no view of the impact of this change on the rest of the code.